bootstrap/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
What is this script for?

Reply:
called in the Makefile

```
debug: push cleanup
        @echo debugging $(IMG):$(TAG) using port $(PORT)
        @./debug.sh $(IMG) $(TAG) $(PORT)
```

Calling `make debug` will build, push the image, cleanup from prior run as 
target dependencies - it will then call debug.sh